### ISSUE TITLE
Debuging problem, *** NameError: name '_pdbcmd_print_frame_status' is not defined

### ISSUE THREAD

**Issue by zorareh** [spyder-ide/spyder#22659]
### What steps will reproduce the problem?

Hi, After upgrading to version 6, I have problem to debug and encounter the below error message:

*** NameError: name '_pdbcmd_print_frame_status' is not defined

Appreciate if you help to resovle this issue.

## Versions

* Spyder version: 6.0.1  (standalone)
* Python version: 3.11.9 64-bit
* Qt version: 5.15.8
* PyQt5 version: 5.15.9
* Operating System: Windows-10-10.0.22631-SP0

### Dependencies

```
# Mandatory:
aiohttp >=3.9.3                  :  3.10.5 (OK)
asyncssh >=2.14.0,<3.0.0         :  2.17.0 (OK)
atomicwrites >=1.2.0             :  1.4.1 (OK)
chardet >=2.0.0                  :  5.2.0 (OK)
cloudpickle >=0.5.0              :  3.0.0 (OK)
cookiecutter >=1.6.0             :  2.6.0 (OK)
diff_match_patch >=20181111      :  20230430 (OK)
github >=2.3.0                   :  2.4.0 (OK)
importlib_metadata >=4.6.0       :  8.5.0 (OK)
intervaltree >=3.0.2             :  3.1.0 (OK)
IPython >=8.13.0,<9.0.0,!=8.17.1 :  8.27.0 (OK)
jedi >=0.17.2,<0.20.0            :  0.19.1 (OK)
jellyfish >=0.7                  :  1.1.0 (OK)
jsonschema >=3.2.0               :  4.23.0 (OK)
keyring >=17.0.0                 :  25.4.1 (OK)
nbconvert >=4.0                  :  7.16.4 (OK)
numpydoc >=0.6.0                 :  1.8.0 (OK)
parso >=0.7.0,<0.9.0             :  0.8.4 (OK)
pexpect >=4.4.0                  :  4.9.0 (OK)
pickleshare >=0.4                :  0.7.5 (OK)
psutil >=5.3                     :  6.0.0 (OK)
pygments >=2.0                   :  2.18.0 (OK)
pylint >=3.1,<4                  :  3.3.0 (OK)
pylint_venv >=3.0.2              :  3.0.3 (OK)
pyls_spyder >=0.4.0              :  0.4.0 (OK)
pylsp >=1.12.0,<1.13.0           :  1.12.0 (OK)
pylsp_black >=2.0.0,<3.0.0       :  2.0.0 (OK)
pyuca >=1.2                      :  1.2 (OK)
qdarkstyle >=3.2.0,<3.3.0        :  3.2.3 (OK)
qstylizer >=0.2.2                :  0.2.3 (OK)
qtawesome >=1.3.1,<1.4.0         :  1.3.1 (OK)
qtconsole >=5.6.0,<5.7.0         :  5.6.0 (OK)
qtpy >=2.4.0                     :  2.4.1 (OK)
rtree >=0.9.7                    :  1.3.0 (OK)
setuptools >=49.6.0              :  74.1.2 (OK)
sphinx >=0.6.6                   :  8.0.2 (OK)
spyder_kernels >=3.0.0,<3.1.0    :  3.0.0 (OK)
superqt >=0.6.2,<1.0.0           :  0.6.7 (OK)
textdistance >=4.2.0             :  4.6.2 (OK)
three_merge >=0.1.1              :  0.1.1 (OK)
watchdog >=0.10.3                :  5.0.2 (OK)
yarl >=1.9.4                     :  1.9.4 (OK)
zmq >=24.0.0                     :  26.2.0 (OK)

# Optional:
cython >=0.21                    :  3.0.11 (OK)
matplotlib >=3.0.0               :  3.9.2 (OK)
numpy >=1.7                      :  2.1.1 (OK)
pandas >=1.1.1                   :  2.2.3 (OK)
scipy >=0.17.0                   :  1.14.1 (OK)
sympy >=0.7.3                    :  1.13.2 (OK)
```

### Environment

<details><summary>Environment</summary>

```
Package           Version
----------------- -----------
asttokens         2.4.1
cloudpickle       3.0.0
colorama          0.4.6
comm              0.2.2
contourpy         1.3.0
cycler            0.12.1
debugpy           1.8.6
decorator         5.1.1
et-xmlfile        1.1.0
executing         2.1.0
fonttools         4.54.1
gmsh              4.13.1
ipykernel         6.29.5
ipython           8.28.0
jedi              0.19.1
jupyter_client    8.6.3
jupyter_core      5.7.2
kiwisolver        1.4.7
matplotlib        3.9.2
matplotlib-inline 0.1.7
MouseInfo         0.1.3
nest-asyncio      1.6.0
numpy             2.1.2
openpyxl          3.1.5
OrcFxAPI          11.4.2
packaging         24.1
pandas            2.2.3
parso             0.8.4
pillow            10.4.0
pip               24.2
platformdirs      4.3.6
prompt_toolkit    3.0.48
psutil            6.0.0
pure_eval         0.2.3
PyAutoGUI         0.9.54
PyGetWindow       0.0.9
Pygments          2.18.0
PyMsgBox          1.0.9
pyparsing         3.1.4
pyperclip         1.9.0
PyRect            0.2.0
PyScreeze         1.0.1
python-dateutil   2.9.0.post0
pytweening        1.2.0
pytz              2024.2
pywin32           307
pyzmq             26.2.0
scipy             1.14.1
six               1.16.0
spyder-kernels    3.0.0
stack-data        0.6.3
tornado           6.4.1
traitlets         5.14.3
tzdata            2024.2
wcwidth           0.2.13

```
</details>

**Comment by ccordoba12** [comment]
Hey @zorareh, thanks for reporting. This is a duplicate of issue #22500, which should be fixed in our 6.0.2 version, to be released in a week and a half.

In the meantime, please downgrade Python to 3.12.4 in your environment as a workaround (that error only happens in Python 3.12.5 or newer).

**Comment by zorareh** [comment]
Hey @ccordoba12 , many thanks for your reply
